Library/Homebrew/test/emoji_spec.rb

```diff
@@ -4,8 +4,15 @@
   describe "#install_badge" do
     subject { described_class.install_badge }
 
-    it "returns 🍺 by default" do
-      expect(subject).to eq "🍺"
+    context "with emoji environment variables unset" do
```

Comment:
Yes, as this is the default, this doesn't really have a context. The `before` block should run before each test, setting this default.